ossindex/ossindex.go

```diff
@@ -55,12 +55,13 @@ func AuditPackages(purls []string) ([]types.Coordinate, error) {
 		for _, purl := range purls {
 			item, err := txn.Get([]byte(strings.ToLower(purl)))
 			if err == nil {
-				val, err := item.Value()
-				if err == nil {
+				err := item.Value(func(val []byte) error {
 					var coordinate types.Coordinate
-					json.Unmarshal(val, &coordinate)
+					err := json.Unmarshal(val, &coordinate)
 					results = append(results, coordinate)
-				} else {
+					return err
```

Comment:
Suggestion: Take a look at using the `customerror` package, if you can't unmarshal the response it might be worth just quitting execution completely.

Reply:
I like the idea, but I'm nervous about changing the actual functionality in this PR 😬

Reply:
YOLO! It's cool tho :)